app/views/description_change_requests/show.html.erb

```diff
@@ -19,34 +16,38 @@
       <p class="govuk-body govuk-!-margin-bottom-1">
         <strong>Request sent:</strong> <%= @description_change_request.created_at.strftime("%e %B %Y") %>
       </p>
-
       <strong class="govuk-!-margin-top-4 govuk-tag govuk-tag--<%=display_request_status(@description_change_request) %>">
         <% if @description_change_request.state == "open" %>
           Open
         <% else %>
           <%= @description_change_request.approved? ? 'Approved' : 'Rejected' %>
         <% end %>
       </strong>
-
       <div class="govuk-!-margin-top-6">
-        <p class="govuk-body"> <strong> Previous description:</strong><br>
-          <%= @planning_application.description  %>
-        </p>
-        <p class="govuk-body"> <strong> Suggested description:</strong><br>
-          <%= @description_change_request.proposed_description  %>
-        </p>
+        <% if @description_change_request.state == "open" || change_rejected?(@description_change_request) %>
+          <p class="govuk-body"> <strong> Previous description:</strong><br>
+            <%= @planning_application.description  %>
+          </p>
+          <p class="govuk-body"> <strong> Suggested description:</strong><br>
+            <%= @description_change_request.proposed_description  %>
+          </p>
+        <% else %>
+          <p class="govuk-body"> <strong> Previous description:</strong><br>
+            <%= @description_change_request.previous_description  %>
+          </p>
+          <p class="govuk-body"> <strong> Suggested description:</strong><br>
+            <%= @description_change_request.proposed_description  %>
+          </p>
```

Comment:
The "Suggested description" is repeated here, it can be removed from the if/else statement.